src/components/header/header.tsx

```diff
@@ -3,74 +3,115 @@ import {
+import { Divider } from "@chakra-ui/react";
 
 const Header = () => {
   useStickyMargin();
-  const isScrolled = useScrolled("header");
+  const isScrolled = useScrolled("header", false);
   return (
     <Box
       data-sticky-header
       data-sticky-offset="10"
       position="sticky"
       id="header"
       top={0}
-      py={{ sm: "xs", md: "s" }}
-      px="s"
       zIndex={999}
       bgColor="ui.white"
+      pt="xs"
     >
       <Box
-        alignItems="center"
         maxWidth="1280px"
-        padding={{ md: "0 15px" }}
         mx="auto"
+        padding="0 16px"
+        sx={{
+          [`@media screen and (min-width: ${headerBreakpoints.smTablet})`]: {
+            display: "block",
+          },
+          [`@media screen and (min-width: ${headerBreakpoints.lgTablet})`]: {
+            display: "flex",
+            justifyContent: "space-between",
+          },
+        }}
       >
         <HStack
+          justify="space-between"
           sx={{
-            justifyContent: "space-between",
-            alignContent: "center",
+            display: "none",
+            [`@media screen and (min-width: ${headerBreakpoints.smTablet})`]: {
+              display: "flex",
+            },
           }}
-          pl="9px"
         >
-          <DCLogo isMobile={true} />
+          <DCLogo isMobile={false} />
           <Box
             sx={{
-              display: { sm: "block", md: "none" },
+              display: "none",
+              [`@media screen and (min-width: ${headerBreakpoints.smTablet})`]:
+                {
+                  display: isScrolled ? "block" : "none",
+                },
+              [`@media screen and (min-width: ${headerBreakpoints.lgTablet})`]:
+                {
+                  display: "none",
+                },
             }}
           >
-            <Text>I am hamburger</Text>
+            <NavMenu />
           </Box>
         </HStack>
-        <HStack justify="space-between">
-          <DCLogo isMobile={false} />
-          <VStack width={{ sm: "100%", md: "36%" }}>
-            <Box
-              sx={{
-                display: {
-                  sm: "none",
-                  md: isScrolled ? "block" : "none",
-                },
-              }}
-            >
-              <Text>I am desktop nav links</Text>
-            </Box>
-            <Box
-              width="100%"
-              pt="s"
-              sx={{
-                display: { sm: isScrolled ? "inline" : "none", md: "inline" },
-              }}
-            >
-              <Search />
-            </Box>
-          </VStack>
+        <HStack
+          sx={{
+            [`@media screen and (min-width: ${headerBreakpoints.smTablet})`]: {
+              display: "none",
+            },
+            justifyContent: "space-between",
+          }}
+        >
+          <DCLogo isMobile={true} />
+          <MobileNavMenu />
         </HStack>
+        <Divider
```

Comment:
The DS has a [`HorizontalRule`](https://nypl.github.io/nypl-design-system/reservoir/v2/?path=/docs/components-page-layout-horizontalrule--docs). Does that work? Mostly so you don't have to import from `"@chakra-ui/react"`.

I think this would be better as a bottom border but the DOM structure doesn't really allow it.